### PR TITLE
Two Tools Added

### PR DESCRIPTION
**Pull request by farif**
Added two new tools "Verifast" and "Sys".

### CODE REVIEW

**Comment by mre** [comment]
Tools look great, thanks! Looks like there are some merge conflicts. Can you reintegrate `master` into your branch? That should take care of it. 😎

**Comment by farif** [comment]
Thanks. I updated the branch. Can you check if the merge conflicts are resolved correctly :)

File: data/tools/verifast.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  - linter
+tags:
+  - ocaml
+license: MIT License
+types:
+  - cli
+source: 'https://github.com/verifast/verifast'
+homepage: 'https://github.com/verifast/verifast'
+description: >- 
+Source Code verification tool for C and Java.
```
Comment (mre) [inline data/tools/verifast.yml:12]: This should be indented. YAML wants it so. 😄

**Comment by mre** [comment]
Hey @farif,
Thanks for reintegrating master. Looks good now.
There is an indentation missing in `veryfast.yml`. I marked the position. That's just a YAML syntax error. 
Would have fixed myself, but it seems like I can't edit your branch. 😄

**Comment by farif** [comment]
All looks good. Thank you for pointing me to the error.

**Comment by mre** [comment]
Merged. Thanks for your contribution!

**Pull request merged**